tasks/noflo_browser.js

```diff
@@ -120,6 +134,13 @@ module.exports = function(grunt) {
             grunt.file.delete(fileOptions.webpack.entry);
           }
           return bluebird.resolve(null);
+        }).then(function () {
+          if (fileOptions.webpack.target === 'node') {
+            // No need to build HTML demos for Node.js
+            return bluebird.resolve(null);
+          }
+          var buildDemos = bluebird.promisify(createDemos);
```

Comment:
Isn't this the regular codepath that everything using grunt-noflo-browser uses? `demos` does not sound like something you'd use to deploy your app with...